build.sbt

```diff
@@ -63,7 +63,7 @@ lazy val scoverageSettings = {
     "uk\\.gov\\.hmrc\\.customs\\.emailfrontend\\.views\\.html\\.partials\\.main_template*",
     "uk\\.gov\\.hmrc\\.customs\\.emailfrontend\\.views\\.html\\.helpers*",
     ".*(BuildInfo|Routes|TestOnly).*").mkString(";"),
-    ScoverageKeys.coverageMinimum := 98.4,
+    ScoverageKeys.coverageMinimum := 98.59,
```

Comment:
I thought we were not doing two numbers after comma anymore?

Reply:
why not? we were saving ourselves from build failures while adding new html `views` but I believe we won't be adding new views anymore or you have different opinion there